libs/pages/elewa/invest/src/lib/pages/elewa-invest-stakeholder-section/elewa-invest-stakeholder-section.component.html

```diff
@@ -0,0 +1,8 @@
+<!-- <p>elewa-invest-stakeholder-section works!</p> -->
+<elewa-group-elewa-group-image-and-text-banner
+[titleText]="title"
+[paragraphTexts]="paragraphs"
+[imageURL]="imageURL"
+[imagePlacement]="imagePosition"
```

Comment:
This component should be placed inside components not pages, then add the page to the invest module main page with it's selector.